course/api/views.py

```diff
@@ -1,4 +1,4 @@
-from rest_framework import filters, generics, permissions, viewsets, status, mixins
+from rest_framework import filters, generics, permissions, viewsets, status, mixins, views
```

Comment:
You added import for `views`, but is it actually used anywhere?

Reply:
It is not, I'll remove it.